tests/Microsoft.Bot.Builder.Core.Extensions.Tests/TranscriptUtilities.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Bot.Schema;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Core.Extensions.Tests
+{
+    public static class TranscriptUtilities
+    {
+        public static IEnumerable<IActivity> GetFromTestContext(TestContext context)
+        {
+            var transcriptsRootFolder = TestUtilities.GetKey("TranscriptsRootFolder") ?? @"..\..\..\..\..\transcripts";
```

Comment:
This code seems a bit fragile. 

The "..\.." here is dependent on build structures (that may change), and the results are not being checked to see if the path actually exists . This will cause (I think) exceptiosn in the File.ReadAllText method when called.

Reply:
We also think the folder structure will change and tried to put the easiest temporal solution in place. The `TestUtilities.GetKey("TranscriptsRootFolder")` looks for an environment variable to override the hardcoded relative path giving us a way to easily patch this.

We might want to discuss where the transcript files (for a cross platform/cross repository approach) should live and yes we'll add some file checking logic.